nubis/puppet/sites.pp

```diff
@@ -37,6 +37,38 @@
   ],
 }
 
+nubis::static { 'local-start':
```

Comment:
It would be nice to have a comment indicating that 'local-start' and 'start-com-redirect' are a paired set, but that's not a strict requirement.

Reply:
I can come back to this. I don't have a good naming convention right now.